strapi/strapi
# [Audit Logs] Don't log events before Strapi is loaded or if there is no user

### What does it do?

In the `processEvent` function on Audit Logs service now we check if Strapi is loaded to log an event and also we check if there is a user executing the action, if not we don't log the event

### Why is it needed?

We are logging the events at a low level, so when Strapi starts it is logging too many permissions events that are not necessary to log, also there are some events without user logged in the audit logs (example: the content events from the API content)

### How to test it?

- Run a Strapi application with a valid EE license
- Create a new content type or edit a existing one and wait for Strapi to restart 
- Go to the audit logs page, you should see only the create/update event and not the delete permissions events
